src/test/java/jewellery/inventory/unit/service/UserServiceTest.java

```diff
@@ -102,36 +97,22 @@ void createUserWhenEmailIsAlreadyTakenThenThrowDuplicateEmailException() {
     verify(userRepository, never()).save(any(User.class));
   }
 
-  @Test
-  @DisplayName("Should throw a DuplicateNameException when the name is already taken")
-  void createUserWhenNameIsAlreadyTakenThenThrowDuplicateNameException() {
-    UserRequestDto userRequest = userMapper.toUserRequest(user);
-
-    when(userRepository.findByName(user.getName())).thenReturn(Optional.of(user));
-    when(userMapper.toUserEntity(userRequest)).thenReturn(user);
-
-    assertThrows(DuplicateNameException.class, () -> userService.createUser(userRequest));
-
-    verify(userRepository, never()).save(any(User.class));
-  }
-
   @Test
   @DisplayName("Should create a new user when the email and name are not taken")
   void createUserWhenEmailAndNameAreNotTaken() {
```

Comment:
Please, change the name of the test to reflect the new requirements. We care if only email is taken now.